src/Common/src/Interop/Windows/SChannel/SslConnectionInfo.cs

```diff
@@ -42,6 +42,7 @@ internal unsafe SslConnectionInfo(byte[] nativeBuffer)
                     {
                         GlobalLog.Assert("SslConnectionInfo::.ctor", "Negative size.");
                     }
+
```

Comment:
We tend to add blank lines after clauses (such as if clauses) since that was/is the StyleCop guidelines for formatting C# code.  This particular formatting is not detailed in the current CoreFx guidelines.  Personally, I do like it as it improves readability.  We might want to consider adding this guidelines for the official CoreFx style guidelines.